### ISSUE TITLE
[Bug] Hellpoint

### ISSUE THREAD

**Issue by originalnicodr** [originalnicodr/CinematicUnityExplorer#55]
### Are you on the latest version of UnityExplorer?

- [X] Yes, I'm on the latest version of UnityExplorer.

### Which release are you using?

BepInEx IL2CPP

### Which game did this occur on?

Hellpoint

### Describe the issue.

Game crashes on UI creation

### Relevant log output

```shell
[20:34:11.197] Support Module Loaded: E:\Games\Hellpoint\MelonLoader\Dependencies\SupportModules\Il2Cpp.dll
[20:34:11.222] [UnityExplorer] UnityExplorer 4.11.1 initializing...
[20:34:11.238] [UnityExplorer] [UniverseLib] UniverseLib 1.5.5 initializing...
[20:34:11.598] [UnityExplorer] [UniverseLib] Finished UniverseLib initial setup.
[20:34:11.653] [UnityExplorer] Initialized UnityCrashPrevention for: Canvas.get_renderingDisplaySize
[20:34:16.322] [UnityExplorer] [UniverseLib] Loaded Unhollowed modules in 4.894 seconds.
[20:34:16.414] [UnityExplorer] [UniverseLib] Setup deobfuscation cache in 0.09 seconds.
[20:34:16.417] [UnityExplorer] [UniverseLib] Initialized Legacy Input support
[20:34:16.518] [UnityExplorer] [UniverseLib] Loaded modern bundle for Unity 2021.1.27f1
[20:34:16.544] [UnityExplorer] [UniverseLib] UniverseLib 1.5.5 initialized.
[20:34:16.547] [UnityExplorer] Creating UI...
mono_class_from_mono_type: implement me 0x00

* Assertion: should not be reached at ..\mono\metadata\class.c:6545
```

**Comment by originalnicodr** [comment]
Seems to be a reflection problem?

https://forum.unity.com/threads/mono_class_from_mono_type-implement-me-0x00.33617/

**Comment by originalnicodr** [comment]
Fixed on 66773b300fac50ca26d22851ce82d11198b4a95d
